src/features/user/UserProfile/components/RedeemModal.tsx

```diff
@@ -27,15 +149,131 @@ export default function RedeemModal({
+                    </p>
+                  </div>
+                </div>
+              </div>
+
+              {/* hidden div for image download */}
+              <div style={{ width: '0px', height: '0px', overflow: 'hidden' }}>
+                <div className={styles.tempThankYouImage} ref={imageRef}>
+                  <div className={styles.tempthankyouImageHeader}>
+                    <p dangerouslySetInnerHTML={{ __html: t('donate:thankyouHeaderText') }} />
+                  </div>
+                  <p className={styles.tempDonationCount}>
+                    My {validCodeData.treeCount} trees will be planted by {validCodeData.tpos[0].tpoName}
+                  </p>
+                  <p className={styles.tempDonationTenant}>
+                    {t('donate:plantTreesAtURL', { url: config.tenantURL })}
+                  </p>
+                </div>
+              </div>
+
+              <div className={styles.shareOptions}>
+                <ShareOptions
+                  treeCount={validCodeData.treeCount}
+                  sendRef={sendRef}
+                  handleTextCopiedSnackbarOpen={handleTextCopiedSnackbarOpen}
+                />
+              </div>
+
+
+              <Snackbar
+                open={textCopiedsnackbarOpen}
+                autoHideDuration={4000}
+                onClose={handleTextCopiedSnackbarClose}
+              >
+                <Alert onClose={handleTextCopiedSnackbarClose} severity="success">
+                  {t('donate:copiedToClipboard')}
+                </Alert>
+              </Snackbar>
+
+            </div>
+          </>
+        ) : (
+            <div className={styles.modal}>
+              {codeValidated && validCodeData ? (
+                <>
+                  <div className={styles.codeTreeCount}>
+                    {validCodeData.treeCount}
+                    <span>trees</span>
+                  </div>
+
+                  <div className={styles.plantedBy}>
+                    <span>Planted by</span>
+                    <p>{validCodeData.tpos[0].tpoName}</p>
+                  </div>
+
+
+                  <div onClick={handleSubmit(redeemCode)} className={styles.continueButton}>
+                    {isUploadingData ? (
+                      <div className={styles.spinner}></div>
+                    ) : (
+                        'Add to my trees'
+                      )}
+                  </div>
+
+
+
+                </>
+              ) : (
+                  <>
+                    <h4>
+                      {t('me:redeem')}
+                    </h4>
+
+                    <div className={styles.inputField}>
+                      <MaterialTextField inputRef={register({
+                        required: {
+                          value: true,
+                          message: 'Please enter a code to redeem.',
+                        }
+                      })} name={'code'} placeholder="pp.eco/XADSA-DS-AS" label="" variant="outlined" />
```

Comment:
Maybe we should change the placeholder `pp.eco/XADSA-DS-AS` to just `XADSA-DS-AS` or something else?

Reply:
we added this as the chocolates will have the link with pp.eco 

@sagararyal